core/scripts/vrfv1/README.md

```diff
@@ -93,7 +93,7 @@ the following steps:
    transferAndCall on the LinkToken contract.
 2. The LinkToken contract transfers funds to the ownerless consumer.
 3. The ownerless consumer requests randomness from the
-   [VRF Coordinator](../../../contracts/src/v0.6/VRFCoordinator.sol), using the
+   [VRF Coordinator](../../../contracts/src/v0.8/vrf/VRFCoordinatorV2.sol), using the
```

Comment:
This is a misleading link, maybe link to https://github.com/smartcontractkit/chainlink-contracts-deprecated/blob/main/contracts/src/v0.6/VRFCoordinator.sol instead since its the right contract?